# Thai translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# Roys Hengwatanakul <roysheng@gmail.com>, 2006.
# Theppitak Karoonboonyanan <thep@linux.thai.net>, 2009.
#
#: ../DistUpgrade/DistUpgradeController.py:1075
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2021-04-13 11:08-0700\n"
"PO-Revision-Date: 2013-05-22 11:00+0000\n"
"Last-Translator: Brian Murray <brian@ubuntu.com>\n"
"Language-Team: Thai <thai-l10n@googlegroups.com>\n"
"Language: th\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:48+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "เซิร์ฟเวอร์สำหรับ %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "เซิร์ฟเวอร์หลัก"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "เซิร์ฟเวอร์กำหนดเอง"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "ไม่สามารถคำนวนรายการใน sources.list ได้"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr "ไม่พบแพคเกจใดเลย บางทีนี่อาจไม่ใช่แผ่น Ubuntu หรือแผ่นอาจเสียหาย?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "ไม่สามารถเพิ่มซีดีได้"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"มีปัญหาในการเพิ่มซีดี การปรับปรุงรุ่นจะถูกยกเลิก กรุณารายงานปัญหานี้ถ้าเป็นซีดีที่ถูกต้องของ Ubuntu\n"
"\n"
"ปัญหาคือ : \n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "ลบแพกเกจส่วนที่ใช้การไม่ได้"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"แพคเกจ '%s' อยู่ในสถานะที่ไม่ถูกต้องและต้องทำการติดตั้งใหม่ แต่ไม่สามารถหาข้อมูลมาแก้ไขได้ "
"คุณต้องการที่จะลบแพคเกจนี้ออกก่อนเพื่อดำเนินการต่อหรือไม่?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "เซิร์ฟเวอร์อาจทำงานหนักเกินไป"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "แพกเกจมีปัญหา"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"ระบบของคุณมีแพกเกจที่มีปัญหาที่ไม่สามารถซ่อมได้ด้วยโปรแกรมนี้ กรุณาซ่อมโดยใช้โปรแกรม synaptic "
"หรือ apt-get ก่อนดำเนินการต่อไป"

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "แพกเกจมีปัญหา"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr "ปัญหานี้ดูเหมือนว่าจะเป็นปัญหาชั่วคราว, กรุณาลองใหม่อีกครั้งภายหลัง"

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "ไม่สามารถคำนวนการปรับปรุงได้"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "ผิดพลาดในการยืนยันของจริงในบางแพกเกจ"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"ไม่สามารถยืนยันความถูกต้องของบางแพกเกจได้ นี่อาจเกี่ยวกับปัญหาด้านเครือข่าย "
"คุณอาจลองอีกครั้งได้ในภายหลัง กรุณาตรวจดูรายการของแพกเกจที่ไม่ผ่านการยืนยัน"

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr "แพกเกจ '%s' ถูกกาไว้เพื่อถอดถอนแต่แพกเกจนี้อยู่ในรายชื่อที่ไม่ให้ลบออก"

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "แพกเกจที่จำเป็น '%s' ถูกกำหนดไว้เพื่อลบออก"

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "พยายามติดตั้งรุ่นที่ถูกขึ้นบัญชีดำ '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "ไม่สามารถติดตั้ง '%s' ได้"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"ไม่สามารถติดตั้งแพกเกจที่ต้องการได้ โปรดรายงานปัญหานี้โดยใช้คำสั่ง 'ubuntu-bug ubuntu-"
"release-upgrader-core' ในเทอร์มินัล"

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "ไม่สามารถเดาเมตาแพกเกจได้"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"ระบบของคุณไม่มีแพกเกจ %s หรือ %s และไม่สามารถตรวจสอบได้ว่าคุณกำลังใช้ Ubuntu รุ่นใด\n"
" โปรดติดตั้งแพกเกจหนึ่งข้างต้นก่อนใช้ synaptic หรือ apt-get ก่อนทำต่อไป"

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "กำลังอ่านแคช"

#: ../DistUpgrade/DistUpgradeController.py:225
msgid "Unable to get exclusive lock"
msgstr "ไม่สามารถเอาล็อคแต่ผู้เดียวได้"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"ปกติแล้วหมายความว่ามีแอปพลิเคชันจัดการแพกเกจอื่น ๆ (เช่น apt-get หรือ aptitude) "
"กำลังทำงานอยู่ กรุณาออกจากโปรแกรมนั้นก่อน"

#: ../DistUpgrade/DistUpgradeController.py:284
msgid "Upgrading over remote connection not supported"
msgstr "ไม่สนับสนุนการปรับรุ่นผ่านการติดต่อระยะไกล"

#: ../DistUpgrade/DistUpgradeController.py:285
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"คุณกำลังเรียกใช้การปรับรุ่นผ่านการเชื่อมต่อระยะไกลด้วย SSH ส่วนติดต่อไม่สนับสนุนการกระทำนี้ "
"โปรดลองโหมดข้อความด้วย 'do-release-upgrade'\n"
"\n"
"การปรับปรุ่นจะยกเลิกเดี๋ยวนี้ โปรดลองโดยไม่ใช้ SSH"

#: ../DistUpgrade/DistUpgradeController.py:299
msgid "Continue running under SSH?"
msgstr "ใช้งานภายใต้ SSH ต่อไปหรือไม่?"

#: ../DistUpgrade/DistUpgradeController.py:300
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"วาระนี้ดูเหมือนจะทำงานภายใต้ SSH ซึ่งไม่แนะนำที่จะทำการอัพเกรดผ่าน SSH "
"เพราะถ้ามีปัญหาเกิดขึ้นมันยากที่จะกู้คืน\n"
"\n"
"ถ้าคุณต้องการทำต่อ ให้เพิ่มที่ดีมอน ssh ด้วยพอร์ต '%s'\n"
"คุณต้องการที่จะต่อหรือไม่"

#: ../DistUpgrade/DistUpgradeController.py:314
msgid "Starting additional sshd"
msgstr "กำลังเพิ่ม sshd อีก"

#: ../DistUpgrade/DistUpgradeController.py:315
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"เพื่อให้การกู้ระบบคืนสู่สภาพเดิมง่ายขึ้นในกรณีที่มีปัญหา โปรแกรมจะเริ่ม sshd อีกอันหนึ่งที่พอร์ต '%s' "
"ถ้ามีปัญหาเกิดขึ้นกับโปรแกรม ssh ที่ใช้อยู่คุณยังสามารถติดต่อผ่านอีกอันหนึ่งได้\n"

#: ../DistUpgrade/DistUpgradeController.py:323
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"ถ้าคุณใช้งานไฟร์วอลล์ คุณควรจะเปิดพอร์ตนี้ชั่วคราว "
"เนื่องจากเป็นสิ่งที่ไม่ปลอดภัยถ้าทำการเปิดพอร์ตทำโดยอัตโนมัติ คุณสามารถเปิดพอร์ตด้วย\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:401
#: ../DistUpgrade/DistUpgradeController.py:425
msgid "Can not upgrade"
msgstr "ไม่สามารถปรับปรุงรุ่น"

#: ../DistUpgrade/DistUpgradeController.py:402
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "เครื่องมือนี้ไม่รองรับการปรับปรุงรุ่นจาก '%s' ไปเป็น '%s'"

#: ../DistUpgrade/DistUpgradeController.py:426
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:455
#, python-format
msgid "Can not write to '%s'"
msgstr "ไม่สามารถเขียนลงใน '%s' ได้"

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"ไม่สามารถเขียนลงในไดเรกทอรีระบบ '%s' บนระบบของคุณ การอัปเกรดไม่สามารถดำเนินการต่อได้\n"
"โปรดตรวจสอบให้แน่ใจว่าไดเรกทอรีระบบสามารถเขียนได้"

#: ../DistUpgrade/DistUpgradeController.py:467
msgid "Include latest updates from the Internet?"
msgstr "รวมการอัปเดตล่าสุดจากอินเทอร์เน็ตด้วยหรือไม่?"

#: ../DistUpgrade/DistUpgradeController.py:468
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"ระบบการอัปเดตสามารถใช้อินเทอร์เน็ตในการดาวน์โหลดการอัปเดตล่าสุดและติดตั้งโดยอัตโนมัติระหว่างที่อัปเกรด "
"หากคุณมีการเชื่อมต่อเครือข่าย ขอแนะนำให้ใช้ตัวเลือกนี้\n"
"\n"
"การอัปเดตจะใช้เวลานานขึ้น แต่เมื่อเสร็จแล้วทุกอย่างในระบบของคุณจะเป็นรุ่นล่าสุดทั้งหมด "
"คุณสามารถที่จะเลือกไม่ทำเช่นนี้ แต่คุณควรจะติดตั้งการอัปเดตโดยเร็วหลังจากที่อัปเกรดเสร็จแล้ว\n"
"หากคุณตอบ 'ไม่' ที่นี่ เครือข่ายจะไม่ถูกใช้เลย"

#: ../DistUpgrade/DistUpgradeController.py:646
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr "ไม่ใช่สำหรับ humans ระหว่างขั้นตอนการพัฒนาของรุ่น %s"

#: ../DistUpgrade/DistUpgradeController.py:747
#, python-format
msgid "disabled on upgrade to %s"
msgstr "หยุดใช้งานเมื่อปรับปรุงเป็น %s"

#: ../DistUpgrade/DistUpgradeController.py:806
msgid "No valid sources.list entry found"
msgstr "ไม่พบรายการ sources.list ที่ใช้ได้"

#: ../DistUpgrade/DistUpgradeController.py:807
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"ไม่พบรายการที่เกี่ยวกับ %s ขณะสแกนที่เก็บของคุณ\n"
"\n"
"การอัปเกรดอาจไม่สำเร็จ\n"
"\n"
"คุณต้องการดำเนินการต่อไปหรือไม่?"

#: ../DistUpgrade/DistUpgradeController.py:817
msgid "No valid mirror found"
msgstr "ไม่เจอเซิรฟ์เวอร์เสริม(mirror)"

#: ../DistUpgrade/DistUpgradeController.py:818
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"ไม่พบรายชื่อแหล่งข้อมูลใดขณะสแกนที่เก็บของคุณ "
"ปัญหานี้อาจเกิดขึ้นหากคุณเรียกใช้แหล่งข้อมูลภายในหรือหากข้อมูลในแหล่งข้อมูลล้าสมัย\n"
"\n"
"คุณต้องการเขียนแฟ้ม 'sources.list' ของคุณใหม่หรือไม่? หากคุณเลือก 'ใช่' "
"ที่นี่จะอัปเกรดรายการ '%s' เป็น '%s' ทั้งหมด\n"
"หากคุณเลือก 'ไม่ใช่' การอัปเกรดจะถูกยกเลิก"

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Generate default sources?"
msgstr "สร้าง default sources?"

#: ../DistUpgrade/DistUpgradeController.py:839
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"หลังจากตรวจแฟ้ม 'sources.list' ของคุณแล้วไม่พบรายการ '%s' ที่ต้องการใช้\n"
"\n"
"ควรเพิ่มรายการ '%s' หรือไม่? ถ้าคุณเลือก 'ไม่' กระบวนการปรับรุ่นจะถูกยกเลิก"

#: ../DistUpgrade/DistUpgradeController.py:874
#: ../DistUpgrade/DistUpgradeController.py:880
msgid "Repository information invalid"
msgstr "ข้อมูลเกี่ยวกับแหล่งเก็บข้อมูลใช้ไม่ได้"

#: ../DistUpgrade/DistUpgradeController.py:875
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"การอัปเกรดข้อมูลในที่เก็บมีผลลัพธ์เป็นแฟ้มที่ไม่ถูกต้อง ดังนั้น กระบวนการรายงานปัญหาจะเริ่มขึ้นทันที"

#: ../DistUpgrade/DistUpgradeController.py:881
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"การอัปเกรดข้อมูลในที่เก็บมีผลลัพธ์เป็นแฟ้มที่ไม่ถูกต้อง เมื่อต้องการรายงานปัญหา ให้ติดตั้ง apport "
"และเรียกใช้ 'apport-bug ubuntu-release-upgrader'"

#: ../DistUpgrade/DistUpgradeController.py:890
msgid "Third party sources disabled"
msgstr "แหล่งอื่นๆใช้ไม่ได้"

#: ../DistUpgrade/DistUpgradeController.py:891
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"บางรายการใน souces.list ของคุณถูกปิดไว้ "
"คุณสามารถเปลี่ยนให้ใช้ได้หลังการปรับปรุงด้วยเครื่องมือ 'software-properties' "
"หรือด้วยโปรแกรมจัดการแพ็กเกจ"

#: ../DistUpgrade/DistUpgradeController.py:933
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "แพคเกจเกิดความขัดแย้ง"

#: ../DistUpgrade/DistUpgradeController.py:936
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"แพคเกจ '%s' อยู่ในสถานะที่ไม่แน่นอนและต้องทำการติดตั้งใหม่ แต่ไม่สามารถหาข้อมูลเก่าได้ "
"กรุณาติดตั้งใหม่อีกครั้งหรือลบออกจากระบบ"

#: ../DistUpgrade/DistUpgradeController.py:999
msgid "Error during update"
msgstr "เกิดข้อผิดพลาดระหว่างที่อัปเดต"

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"มีปัญหาเกิดขึ้นระหว่างที่อัปเดต โดยปกติจะเป็นปัญหาเกี่ยวกับเครือข่าย "
"โปรดตรวจสอบการเชื่อมต่อเครือข่ายของคุณแล้วลองใหม่อีกครั้ง"

#: ../DistUpgrade/DistUpgradeController.py:1043
#: ../DistUpgrade/DistUpgradeController.py:1048
msgid "EFI System Partition (ESP) not usable"
msgstr "ไม่สามารถใช้พาร์ทิชันระบบแบบ EFI (ESP) ได้"

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"พาร์ทิชันระบบแบบ EFI (ESP) ของคุณไม่ได้ถูกเมานท์ไว้ที่ /boot/efi "
"โปรดตรวจสอบให้แน่ใจว่าคุณได้กำหนดค่าอย่างถูกต้องแล้วลองอีกครั้ง"

#: ../DistUpgrade/DistUpgradeController.py:1049
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"พาร์ทิชันะบบแบบ EFI (ESP) ที่เมานท์ไว้ที่ /boot/efi ไม่สามารถเขียนได้ "
"โปรดเมานท์พาร์ทิชันนี้เป็นแบบอ่านและเขียนแล้วลองอีกครั้ง"

#: ../DistUpgrade/DistUpgradeController.py:1056
msgid "Not enough free disk space"
msgstr "ไม่มีพื้นที่ว่างในดิสก์เพียงพอ"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1059
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"การอัปเกรดถูกยกเลิก การอัปเกรดจำเป็นต้องมีพื้นที่ว่างบนดิสก์ '{str_dir}' อย่างน้อย "
"{str_total} กรุณาเพิ่มพื้นที่ว่างบนดิสก์ '{str_dir}' อย่างน้อย {str_needed} \r\n"
"{str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1064
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr "เอาแพกเกจชั่วคราวจากการติดตั้งครั้งก่อนหน้าออกโดยใช้ 'sudo apt clean'"

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"คุณสามารถเอาเคอร์เนลเก่าออกได้โดยใช้คำสั่ง 'sudo apt autoremove' และคุณก็สามารถตั้งค่า "
"COMPRESS=xz ใน /etc/initramfs-tools/initramfs.conf เพื่อปรับลดขนาด initramfs "
"ของคุณได้"

#: ../DistUpgrade/DistUpgradeController.py:1071
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"เทขยะของคุณและเอาแพกเกจชั่วคราวจากการติดตั้งครั้งก่อนหน้าออกโดยใช้คำสั่ง 'sudo apt-get "
"clean'"

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid "Reboot to clean up files in /tmp."
msgstr "รีบูตเพื่อทำความสะอาดแฟ้มใน /tmp"

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1113
#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2047
msgid "Calculating the changes"
msgstr "กำลังคำนวนปริมาณการเปลี่ยนแปลง"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1157
msgid "Do you want to start the upgrade?"
msgstr "คุณต้องการที่จะเริ่มการปรับปรุงหรือไม่?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""
"การอัปเดตด้านความปลอดภัยของ Livepatch ไม่พร้อมใช้งานสำหรับ Ubuntu %s หากคุณอัปเกรด "
"Livepatch จะปิดการทำงาน"

#: ../DistUpgrade/DistUpgradeController.py:1248
msgid "Upgrade canceled"
msgstr "การปรับรุ่นถูกยกเลิก"

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr "การปรับรุ่นจะถูกยกเลิกเดี๋ยวนี้ และระบบดังเดิมจะถูกกู้คืน คุณสามารถปรับรุ่นต่อได้ในภายหลัง"

#: ../DistUpgrade/DistUpgradeController.py:1255
#: ../DistUpgrade/DistUpgradeController.py:1400
msgid "Could not download the upgrades"
msgstr "ไม่สามารถดาวน์โหลดข้อมูลปรับปรุงรุ่น"

#: ../DistUpgrade/DistUpgradeController.py:1256
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"การปรับรุ่นถูกยกเลิก โปรดตรวจดูการเชื่อมต่ออินเทอร์เน็ตของคุณหรือการติดตั้งสื่อแล้วลองใหม่ ทุก ๆ "
"แฟ้มที่ดาว์โหลดจะถูกเก็บไว้ก่อน"

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1349
#: ../DistUpgrade/DistUpgradeController.py:1388
#: ../DistUpgrade/DistUpgradeController.py:1504
msgid "Error during commit"
msgstr "เกิดข้อผิดพลาดขณะแก้ไขปรับปรุง"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1351
#: ../DistUpgrade/DistUpgradeController.py:1390
#: ../DistUpgrade/DistUpgradeController.py:1551
msgid "Restoring original system state"
msgstr "คืนระบบสู่สถานะแรกเริ่ม"

#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1367
#: ../DistUpgrade/DistUpgradeController.py:1391
msgid "Could not install the upgrades"
msgstr "ไม่สามารถปรับปรุงได้"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1357
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"การอัปเกรดถูกยกเลิก ระบบของคุณอาจอยู่ในสถานะที่ใช้การไม่ได้ การกู้คืนจะเรียกใช้ตอนนี้ (dpkg --"
"configure -a)"

#: ../DistUpgrade/DistUpgradeController.py:1362
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"โปรดรายงานปัญหานี้ โดยเปิดเบราว์เซอร์แล้วเข้าไปที่ http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug และแนบแฟ้ม /var/log/dist-upgrade/ "
"เพื่อรายงานปัญหา\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1401
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr "การปรับรุ่นถูกเพิกเฉย โปรดตรวจสอบการเชื่อมต่ออินเทอร์เน็ต หรือสื่อที่ใช้ติดตั้งและลองใหม่ "

#: ../DistUpgrade/DistUpgradeController.py:1429
msgid "Searching for obsolete software"
msgstr "ค้นหาซอฟแวร์ที่ล้าสมัย"

#: ../DistUpgrade/DistUpgradeController.py:1492
msgid "Remove obsolete packages?"
msgstr "ลบแพจเกจที่ล้าสมัยทิ้งหรือไม่?"

#: ../DistUpgrade/DistUpgradeController.py:1493
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "คงไว้"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "_Remove"
msgstr "เอาออก"

#: ../DistUpgrade/DistUpgradeController.py:1505
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "มีปัญหาเกิดขึ้นขณะทำการเก็บกวาด กรุณาอ่านข้อมูลข้างล่างสำหรับรายละเอียด "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Required depends is not installed"
msgstr "โปรแกรมที่ต้องการไม่ได้ถูกติดตั้ง"

#: ../DistUpgrade/DistUpgradeController.py:1590
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "โปรแกรม '%s' ที่ต้องการไม่ได้ถูกติดตั้ง "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1857
#: ../DistUpgrade/DistUpgradeController.py:1947
msgid "Checking package manager"
msgstr "กำลังตรวจสอบโปรแกรมจัดการแพกเกจ"

#: ../DistUpgrade/DistUpgradeController.py:1863
#: ../DistUpgrade/DistUpgradeController.py:1869
msgid "Preparing the upgrade failed"
msgstr "การเตรียมการปรับปรุงรุ่นมีปัญหา"

#: ../DistUpgrade/DistUpgradeController.py:1864
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr "การเตรียมการอัปเกรดระบบล้มเหลว ดังนั้น กระบวนการรายงานปัญหาจะเริ่มขึ้น"

#: ../DistUpgrade/DistUpgradeController.py:1870
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"การเตรียมการอัปเกรดระบบล้มเหลว เมื่อต้องการรายงานปัญหา ให้ติดตั้ง apport หลังจากนั้น "
"เรียกใช้คำสั่ง 'apport-bug ubuntu-release-upgrader'"

#: ../DistUpgrade/DistUpgradeController.py:1887
#: ../DistUpgrade/DistUpgradeController.py:1897
msgid "Getting upgrade prerequisites failed"
msgstr "เอาแพคเกจที่จำเป็นก่อนการปรับปรุงมาไม่ได้"

#: ../DistUpgrade/DistUpgradeController.py:1888
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"ระบบไม่สามารถรับข้อมูลเกี่ยวกับข้อกำหนดเบื้องต้นในการปรับรุ่นได้ "
"การอัปเกรดจะถูกยกเลิกในขณะนี้และสภาพของระบบจะถูกคืนค่ากลับไปเป็นค่าดั้งเดิม\n"
"\n"
"นอกจากนี้ กระบวนการรายงานปัญหาก็จะเริ่มขึ้นด้วยเช่นกัน"

#: ../DistUpgrade/DistUpgradeController.py:1898
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"ระบบไม่สามารถรับข้อมูลเกี่ยวกับข้อกำหนดเบื้องต้นในการปรับรุ่นได้ "
"การอัปเกรดจะถูกยกเลิกในขณะนี้และสภาพของระบบจะถูกคืนค่ากลับไปเป็นค่าดั้งเดิม\n"
"\n"
"เมื่อต้องการรายงานปัญหา ให้ติดตั้ง apport และหลังจากนั้น ให้เรียกใช้คำสั่ง 'apport-bug "
"ubuntu-release-upgrader'"

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Updating repository information"
msgstr "ปรับปรุงข้อมูลของแหล่งข้อมูล"

#: ../DistUpgrade/DistUpgradeController.py:1938
msgid "Failed to add the cdrom"
msgstr "เพิ่มแผ่นซีดีล้มเหลว"

#: ../DistUpgrade/DistUpgradeController.py:1939
msgid "Sorry, adding the cdrom was not successful."
msgstr "ขออภัยด้วย, ทำการเพิ่มแผ่นซีดีไม่สำเร็จ"

#: ../DistUpgrade/DistUpgradeController.py:1974
msgid "Invalid package information"
msgstr "ข้อมูลของแพกเกจไม่ถูกต้อง"

#: ../DistUpgrade/DistUpgradeController.py:1975
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"หลังจากอัปเดตข้อมูลแพกเกจของคุณแล้ว แพกเกจสำคัญ '%s' ไม่สามารถตั้งขึ้นได้ "
"ปัญหานี้อาจเกิดขึ้นเนื่องจากคุณไม่มีแหล่งข้อมูลทางการในรายชื่อแหล่งข้อมูลซอฟต์แวร์ของคุณ "
"หรือเนื่องจากมีการโหลดข้อมูลจากแหล่งข้อมูลที่คุณกำลังใช้มากเกินไป ดูที่ /etc/apt/sources.list "
"สำหรับรายชื่อปัจจุบันของแหล่งข้อมูลซอฟต์แวร์ที่กำหนดค่าอยู่เดิมแล้ว\n"
"ในกรณีที่มีการโหลดข้อมูลจากแหล่งข้อมูลมากเกินไป "
"คุณอาจต้องการทดลองการอัปเกรดใหม่อีกครั้งในภายหลัง"

#: ../DistUpgrade/DistUpgradeController.py:2004
#: ../DistUpgrade/DistUpgradeController.py:2103
msgid "Fetching"
msgstr "กำลังดึงข้อมูล"

#: ../DistUpgrade/DistUpgradeController.py:2011
#: ../DistUpgrade/DistUpgradeController.py:2032
#: ../DistUpgrade/DistUpgradeController.py:2053
#: ../DistUpgrade/DistUpgradeController.py:2107
msgid "Upgrading"
msgstr "กำลังปรับปรุง"

#: ../DistUpgrade/DistUpgradeController.py:2013
msgid "Upgrade infeasible"
msgstr "อัปเกรดไม่ได้"

#: ../DistUpgrade/DistUpgradeController.py:2014
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr "ไม่สามารถดำเนินการอัปเกรดให้เสร็จสมบูรณ์ได้ เนื่องจากมีข้อผิดพลาดระหว่างการอัปเกรด"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2035
msgid "Upgrade incomplete"
msgstr "การอัปเกรดไม่เสร็จสมบูรณ์"

#: ../DistUpgrade/DistUpgradeController.py:2036
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr "การอัปเกรดเสร็จสมบูรณ์เพียงบางส่วน แต่มีข้อผิดพลาดระหว่างการอัปเกรด"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2058
#: ../DistUpgrade/DistUpgradeController.py:2109
#: ../DistUpgrade/DistUpgradeController.py:2115
#: ../DistUpgrade/DistUpgradeController.py:2126
msgid "Upgrade complete"
msgstr "ปรับปรุงเสร็จแล้ว"

#: ../DistUpgrade/DistUpgradeController.py:2059
#: ../DistUpgrade/DistUpgradeController.py:2110
#: ../DistUpgrade/DistUpgradeController.py:2116
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "การปรับรุ่นสมบูรณ์ แต่เกิดข้อผิดพลาดในระหว่างขั้นตอนการปรับรุ่น"

#: ../DistUpgrade/DistUpgradeController.py:2078
msgid "System upgrade is complete."
msgstr "ปรับปรุงระบบเสร็จแล้ว"

#: ../DistUpgrade/DistUpgradeController.py:2127
msgid "The partial upgrade was completed."
msgstr "การปรับปรุงรุ่นบางส่วนเสร็จสมบูรณ์"

#: ../DistUpgrade/DistUpgradeController.py:2155
msgid "Lock screen disabled"
msgstr "ปิดใช้งานหน้าจอล็อกแล้ว"

#: ../DistUpgrade/DistUpgradeController.py:2156
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr "หน้าจอล็อกของคุณจะถูกปิดใช้งานจนกว่าคุณจะบูตเครื่องใหม่"

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "ไม่สามารถหาบันทึกการปล่อย"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "เซิรฟ์เวอร์อาจจะทำงานเกินพิกัด "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "ไม่สามารถดาวน์โหลดบันทึกการปล่อย"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "กรุณาตรวจสอบการสื่อสารทางอินเตอร์เน็ตของคุณ"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "ตรวจสอบความถูกต้องของ '%(file)s' ผ่าน '%(signature)s' "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:99
#, python-format
msgid "extracting '%s'"
msgstr "กำลังแตก '%s'"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
#: ../DistUpgrade/DistUpgradeFetcherCore.py:121
msgid "Could not run the upgrade tool"
msgstr "ไม่สามารถเรียกใช้เครื่องมือปรับปรุง"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:122
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"ปัญหานี้ดูเหมือนว่าจะเป็นปัญหาเกี่ยวกับเครื่องมืออัปเกรดมากกว่า กรุณารายงานปัญหานี้ในฐานะ bug "
"โดยใช้คำสั่ง 'ubuntu-bug ubuntu-release-upgrader-core'"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:199
msgid "Upgrade tool signature"
msgstr "รายเซ็นของเครื่องมือปรับปรุง"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:206
msgid "Upgrade tool"
msgstr "เครื่องมือปรับปรุงรุ่น"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:240
msgid "Failed to fetch"
msgstr "ไม่สามารถเอามาได้"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:241
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "ไม่สามารถดาวน์โหลดข้อมูลปรับปรุงได้ อาจจะเป็นปัญหาในเครือข่าย "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:245
msgid "Authentication failed"
msgstr "ไม่สามารถยืนยันว่าเป็นของจริงได้"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:246
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr "ยืนยันการปรับปรุงไม่สำเร็จ อาจจะมีปัญหาในเครือข่ายหรือที่เซิรฟ์เวอร์ "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:251
msgid "Failed to extract"
msgstr "ไม่สามารถเอาออกมาได้"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:252
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr "ไม่สามารถแกะข้อมูลปรับปรุงออกมาได้ อาจจะเป็นปัญหาในเครือข่ายหรือที่เซิรฟ์เวอร์ "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:257
msgid "Verification failed"
msgstr "การตรวจสอบล้มเหลว"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:258
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr "ตรวจทานการปรับปรุงไม่สำเร็จ อาจจะมีปัญหาในเครือข่ายหรือที่เซิร์ฟเวอร์ "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:272
#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid "Can not run the upgrade"
msgstr "ไม่สามารถเริ่มโปรแกรมปรับปรุงได้"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"นี้มักจะเกิดโดยที่ระบบ /tmp เชื่อมต่อด้วย noexec โปรดเชื่อมต่อใหม่โดยไม่ต้องใช้ noexec "
"และเรียกใช้ปรับรุ่นอีกครั้ง"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
#, python-format
msgid "The error message is '%s'."
msgstr "ข้อผิดพลาดคือ '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&อัปเกรด"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "&ยกเลิก"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "บันทึกรายการปรับปรุง"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "ปรับปรุง"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "กำลังดาวน์โหลดไฟล์ของแพคเกจเพิ่มเติม..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "ไฟล์ที่ %s จากทั้งหมด %s ด้วยความเร็ว %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "ไฟล์ที่ %s จากทั้งหมด %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "โปรดใส่ '%s' ลงใน '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "เปลี่ยนสื่อ"

#: ../DistUpgrade/DistUpgradeQuirks.py:226
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr "ฮาร์ดแวร์แสดงผลของคุณอาจไม่ได้รับการสนับสนุนอย่างเต็มที่ใน Ubuntu 14.04"

#: ../DistUpgrade/DistUpgradeQuirks.py:228
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"การเรียกใช้สภาพแวดล้อมเดสก์ท็อปของ 'unity' "
"จะไม่ได้รับการสนับสนุนอย่างเต็มที่โดยฮาร์ดแวร์แสดงผลของคุณ "
"คุณอาจพบกับสภาพแวดล้อมการทำงานที่ช้ามากหลังการปรับรุ่น คำแนะนำของเรา คือ ให้ใช้รุ่น LTS "
"ไปก่อนตอนนี้ สำหรับข้อมูลเพิ่มเติม ดูที่ https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D คุณยังคงต้องการอัปเกรดต่อไปหรือไม่?"

#: ../DistUpgrade/DistUpgradeQuirks.py:252
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "ฮาร์ดแวร์แสดงผลของคุณอาจไม่ได้รับการสนับสนุนอย่างเต็มที่ใน Ubuntu 12.04 LTS"

#: ../DistUpgrade/DistUpgradeQuirks.py:254
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"การสนับสนุนใน Ubuntu 12.04 LTS สำหรับฮาร์ดแวร์แสดงผล Intel "
"ของคุณถูกจำกัดและคุณอาจประสบปัญหาหลังการปรับรุ่นได้ สำหรับข้อมูลเพิ่มเติม ดูที่ https://wiki."
"ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx คุณต้องการทำการอัปเกรดต่อไปหรือไม่?"

#: ../DistUpgrade/DistUpgradeQuirks.py:274
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"การปรับปรุงอาจจะลดลูกเล่นของพื้นโต๊ะ และประสิทธิภาพในเกมและโปรแกมอื่นๆ ที่ต้องใช้กราฟิกมาก"

#: ../DistUpgrade/DistUpgradeQuirks.py:278
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"คอมพิวเตอร์ตัวนี้ใช้กราฟิกไดรเวอร์ของ NVIDIA 'nvidia' "
"ไม่พบรุ่นของไดรเวอร์ที่ทำงานกับการ์ดวิดีโอของคุณใน Ubuntu 10.04 LTS\n"
"\n"
"คุณต้องการทำต่อ?"

#: ../DistUpgrade/DistUpgradeQuirks.py:317
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:318
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:333
msgid "No ARMv6 CPU"
msgstr "ไม่พบ CPU ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:334
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"ระบบของคุณใช้ CPU แบบ ARM ซึ่งเก่ากว่าสถาปัตยกรรม ARMv6 แพกเกจทั้งหมดใน karmic "
"จะถูกสร้างขึ้นด้วยการเพิ่มประสิทธิภาพซึ่งต้องการ ARMv6 เป็นสถาปัตยกรรมขั้นต่ำสุด "
"และเป็นไปไม่ได้ที่จะอัปเกรดระบบของคุณเป็น Ubuntu รุ่นใหม่ด้วยฮาร์ดแวร์นี้"

#: ../DistUpgrade/DistUpgradeQuirks.py:354
msgid "No init available"
msgstr "ไม่พบ init"

#: ../DistUpgrade/DistUpgradeQuirks.py:355
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"ระบบของคุณดูเหมือนจะอยู่ในสภาพแวดล้อมแบบจำลองซึ่งไม่มีดีมอน init เช่น Linux-VServer "
"Ubuntu 10.04 LTS ไม่สามารถทำงานโดยใช้สภาพแวดล้อมประเภทนี้ได้ "
"ซึ่งต้องอัปเดตค่ากำหนดของเครื่องเสมือนของคุณก่อน\n"
"\n"
"คุณแน่ใจหรือว่าคุณต้องการทำต่อไป?"

#: ../DistUpgrade/DistUpgradeQuirks.py:396
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:397
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:515
#, fuzzy
msgid "Connection to the Snap Store failed"
msgstr "การเชื่อมต่อไปยัง Snap Store ล้มเหลว"

#: ../DistUpgrade/DistUpgradeQuirks.py:516
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:531
msgid "Connection to Snap Store failed"
msgstr "การเชื่อมต่อไปยัง Snap Store ล้มเหลว"

#: ../DistUpgrade/DistUpgradeQuirks.py:532
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"ระบบของคุณไม่มีการเชื่อมต่อไปยัง Snap Store เพื่อประสบการณ์การอัปเกรดที่ดีที่สุด "
"ตรวจให้แน่ใจว่าระบบของคุณสามารถเชื่อมต่อไปยัง  api.snapcraft.io ได้\n"
"คุณยังคงต้องการดำเนินการอัปเกรดต่อหรือไม่?"

#: ../DistUpgrade/DistUpgradeQuirks.py:541
msgid "Outdated snapd package"
msgstr "แพ็กเกจ snapd ล้าสมัย"

#: ../DistUpgrade/DistUpgradeQuirks.py:542
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"ระบบของคุณไม่มี snapd เวอร์ชันล่าสุด โปรดอัปเดตเวอร์ชันของ snapd "
"บนระบบของคุณเพื่อปรับปรุงประสบการณ์การอัปเกรด\n"
"คุณยังคงต้องการดำเนินการอัปเกรดต่อหรือไม่?"

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:565
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:596
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:602
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:606
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:609
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:638
msgid "PAE not enabled"
msgstr "ไม่ได้เปิดใช้งาน PAE อยู่"

#: ../DistUpgrade/DistUpgradeQuirks.py:639
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"ระบบของคุณใช้ CPU ที่ไม่ได้เปิดใช้งาน PAE อยู่ Ubuntu สนับสนุนเฉพาะระบบที่ไม่ใช่ PAE ตั้งแต่ "
"Ubuntu 12.04 เท่านั้น เมื่อต้องการอัปเกรดเป็น Ubuntu รุ่นถัดไป คุณต้องเปิดใช้งาน PAE "
"(หากเป็นไปได้) ดูที่:\n"
"http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:1000
msgid "Checking for installed snaps"
msgstr "กำลังตรวจหา snaps ที่ติดตั้งแล้ว"

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "ใช้พาธ(path)ที่ให้ในการค้นหาซีดีรอมที่มีแพ็กเกจที่สามารถปรับปรุงได้"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"ใช้ส่วนหน้า(frontend) ปัจจุบันมี่ให้เลือก: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*ไม่สนับสนุนให้ใช้* ตัวเลือกนี้จะไม่ถูกสนใจ"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "ทำการปรับปรุงบางส่วนเท่านั้น (sources.list จะไม่ถูกบันทึก)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "ปิดการทำงานหน้าจอ GNU"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "ตั้งค่า datadir"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "อัปเกรดเป็นรุ่นการพัฒนา"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "ดึงข้อมูลเสร็จสิ้นแล้ว"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "กำลังเอาไฟล์ที่ %li จากทั้งหมด %li ด้วยความเร็ว %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "เหลือประมาณ %s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "กำลังดึงข้อมูลแฟ้ม %li จาก %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "กำลังเปลี่ยนแปลง"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "มีปัญหาความขึ้นต่อกันระหว่างแพกเกจ - จะทิ้งไว้โดยไม่ตั้งค่า"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "ไม่สามารถติดตั้ง '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"การปรับรุ่นจะดำเนินการต่อไป แต่แพคเกจ '%s' อาจจะไม่สามารถทำงานได้ "
"โปรดตรวจสอบและส่งรายการข้อผิดพลาดที่เกี่ยวกับมัน"

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"เปลี่ยนไฟล์ปรับแต่งที่แก้ไขเอง\n"
"'%s' หรือไม่?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr "คุณจะสูญเสียข้อมูลปรับแต่งที่ได้ทำไว้ในไฟล์ปรับแต่งถ้าคุณเลือกที่จะเปลี่ยนไปใช้ไฟล์รุ่นใหม่กว่านี้"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "ไม่เจอคำสั่ง  'diff'"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "เกิดข้อผิดพลาดอย่างร้ายแรง"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"โปรดรายงานปัญหานี้ในฐานะ bug (หากคุณยังไม่ได้ทำ) และแนบแฟ้ม /var/log/dist-upgrade/"
"main.log and /var/log/dist-upgrade/apt.log ในการรายงานของคุณ "
"การอัปเกรดถูกยกเลิก\n"
"sources.list ดั้งเดิมของคุณจะถูกบันทึกใน /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Ctrl-c ถูกกดแล้ว"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"การกดนี้จะยกเลิกการทำงานของโปรแกรม และอาจทำให้ระบบของคุณอยู่ในสภาพใช้การไม่ได้ "
"คุณแน่ใจหรือไม่ว่าจะหยุด?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr "เพื่อป้องกันข้อมูลสูญหาย โปรดออกจากทุกแอปพลิเคชันและเอกสารที่เปิดอยู่"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "เลิกให้การสนับสนุนโดย Canonical แล้ว (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>ปรับรุ่นลง (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "ถอดถอน (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "ไม่ต้องการอีกต่อไป (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "ติดตั้ง (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "ปรับรุ่น (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "แสดงความแตกต่าง >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< ซ่อนความแตกต่าง"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "ผิดพลาด"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&ปิด"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "แสดงเทอร์มินัล >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< ซ่อนเทอร์มินัล"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "รายละเอียด"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&เริ่มการอัปเกรด"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "รายละเอียด"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "เลิกให้การสนับสนุนแล้ว %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "ลบ %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "ถอดถอน %s (ที่ถูกติดตั้งโดยอัตโนมัติ)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "ติดตั้ง  %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "ปรับปรุงรุ่น %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "ต้องการเริ่มระบบใหม่"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>เริ่มระบบใหม่เพื่อที่จะให้การปรับปรุงเสร็จสิ้น</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "เริ่มระบบใหม่เดี๋ยวนี้"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>ยกเลิกการปรับปรุงนี้หรือไม่?</big></b>\n"
"\n"
"ระบบของคุณจะอยู่ในสภาพที่ใช้การไม่ได้ถ้าคุณยกเลิกการปรับปรุง. ขอแนะนำให้ดำเนินการต่อไป"

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "ยกเลิกการปรับปรุงรุ่นหรือไม่?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li วัน"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li ชั่วโมง"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li นาที"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li วินาที"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"ดาวน์โหลดนี้จะใช้เวลาประมาณ %s ถ้าใช้สาย 1Mbit DSL หรือประมาณ %s ถ้าใช้ 56k โมเด็ม."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "การดาวน์โหลดนี้จะใช้เวลาประมาณ %s จากการเชื่อมต่อของคุณ "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "กำลังเตรียมการปรับปรุง"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "ตรวจหาช่องซอฟต์แวร์ใหม่"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "กำลังรับแพคเกจใหม่"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "กำลังทำการปรับปรุงรุ่น"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "กำลังเก็บกวาด"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d แพกเกจที่ติดตั้งแล้วนี้ Canonical ไม่ได้ดูแลอีกต่อไป "
"แต่คุณยังได้รับการสนับสนุนจากชุมชนอื่น ๆ ได้อีก"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d แพกเกจจะถูกลบออก"

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d แพกเกจใหม่จะถูกติดตั้ง"

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d แพกเกจจะถูกปรับปรุงรุ่น"

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"คุณจะต้องดาวน์โหลดทั้งหมด  %s "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"การติดตั้งอัปเกรดอาจใช้เวลาหลายชั่วโมง เมื่อการดาวน์โหลดเสร็จแล้ว "
"คุณไม่สามารถยกเลิกการทำงานได้"

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"การรับข้อมูลและติดตั้งอัปเกรดอาจใช้เวลาหลายชั่วโมง เมื่อการดาวน์โหลดเสร็จแล้ว "
"คุณไม่สามารถยกเลิกการทำงานได้"

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "การถอดถอนแพกเกจอาจใช้เวลาหลายชั่วโมง "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "ซอฟต์แวร์บนคอมพิวเตอร์นี้เป็นรุ่นล่าสุดแล้ว"

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "ไม่มีข้อมูลปรับปรุงรุ่นสำหรับระบบของคุณ การปรับปรุงรุ่นจึงถูกยกเลิก"

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "ต้องเริ่มระบบใหม่"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "การปรับปรุงเสร็จสิ้นแล้วและต้องการที่จะเริ่มระบบใหม่ คุณต้องการที่จะทำเดี๋ยวนี้หรือเปล่า?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "ความคืบหน้า: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"โปรดรายงานปัญหานี้และแนบแฟ้ม /var/log/dist-upgrade/main.log and /var/log/dist-"
"upgrade/apt.log มาในรายงานของคุณด้วย การอัปเกรดถูกยกเลิก\n"
"แฟ้ม sources.list รุ่นเก่าของคุณได้ถูกบันทึกใน  /etc/apt/sources.list.distUpgrade แล้ว"

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "กำลังยกเลิก"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "ปรับลง:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "ทำต่อโปรดกด [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "ดำเนินการต่อไป[ชม] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "รายละเอียด[ร]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "ช"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "ม"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "ร"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "เลิกให้การสนับสนุนแล้ว: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "ลบออก: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "ติดตั้ง:  %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "ปรับปรุง: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "ทำต่อไป [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"คุณจำเป็นต้องเริ่มระบบใหม่เพื่อทำให้การอัพเกรดเสร็จสมบูรณ์\n"
"หากคุณเลือก 'y' จะเป็นการเริ่มระบบใหม่"

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr ""
"กำลังดาวน์โหลดไฟล์ที่ %(current)li จากทั้งหมด %(total)li ด้วยความเร็ว %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "กำลังดาวน์โหลดไฟล์ที่ %(current)li จากทั้งหมด %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "แสดงความคืบหน้าของแแต่ละแฟ้ม"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "ยกเลิกการปรับปรุงรุ่น"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "ปรับปรุงรุ่นต่อจากคราวที่แล้ว"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>ต้องการยกเลิกการปรับปรุงที่กำลังทำอยู่หรือไม่?</big></b>\n"
"\n"
"ระบบอาจจะไม่มั่นคงถ้าคุณยกเลิกการปรับปรุง ขอแนะนำเป็นอย่างยิ่งให้คุณดำเนินการปรับปรุงต่อไป"

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "เริ่มการปรับปรุงรุ่น"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "แทนที่"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "ข้อแตกต่างระหว่างไฟล์"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "ส่งรายงานปัญหา"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "ดำเนินการต่อไป"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>เริ่มการปรับปรุงหรือไม่?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>เริ่มระบบใหม่เพื่อให้การปรับรุ่นสมบูรณ์</big></b>\n"
"โปรดบันทึกงานของท่านก่อนดำเนินการต่อ"

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "ปรับปรุงชุดเผยแพร่"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "กำลังตั้งค่าช่องซอฟต์แวร์ใหม่"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "เริ่มเครื่องคอมพิวเตอร์ใหม่"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "เทอร์มินัล"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "ปรับปรุงรุ่น"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>สามารถปรับรุ่นเป็น Ubuntu รุ่นใหม่ได้แล้ว คุณต้องการปรับรุ่นเลยหรือไม่?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "ไม่ต้องปรับรุ่น"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "ถามฉันในครั้งหน้า"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "ใช่ ดำเนินการปรับรุ่นตอนนี้"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "คุณได้ตัดสินใจไม่ปรับรุ่น Ubuntu เป็นรุ่นใหม่"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"คุณสามารถทำการอัปเกรดได้ในภายหลังโดยเปิด โปรแกรมอัปเดตซอฟต์แวร์ และคลิก \"อัปเกรด\""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "ดำเนินการอัปเกรดแบบทั้งหมด"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "คุณต้องยืนยันตัวตนเพื่อทำการอัปเกรด Ubuntu"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "ดำเนินการอัปเกรดแบบบางส่วน"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "คุณต้องยืนยันตัวตนเพื่อทำการอัปเกรดแบบบางส่วน"

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "แสดงรุ่นและออกจากโปรแกรม"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "ไดเรกทอรีมีแฟ้มข้อมูล"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "เริ่มส่วนหน้า(frontend)ที่กำหนดไว้"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "เริ่มการปรับปรุงรุ่นบางส่วน"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "กำลังดาวน์โหลดเครื่องมือปรับรุ่น"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr "ถ้าใช้รุ่นที่สนับสนุนล่าสุดอยู่ อัปเกรดเป็นรุ่นการพัฒนา"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr "ทดลองอัพเกรดเป็นรุ่นล่าสุดด้วยตัวอัปเกรดจาก $distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"ใช้งานโหมดการอัพเกรดแบบพิเศษ\n"
"ขณะนี้สามารถใช้ 'desktop' สำหรับการอัปเกรดปกติของระบบ Desktop และ 'server' "
"สำหรับระบบแม่ขาย"

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr "ตรวจสอบเฉพาะตอนมีรุ่นใหม่ออกและรายงานผลผ่านทางรหัสออก"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "ตัวเลือก --devel-release และ --proposed"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr "เป็นตัวเลือกพิเศษเหมือนกัน โปรดเลือกใช้เพียงตัวเลือกเดียว"

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "กำลังตรวจสอบ Ubuntu รุ่นใหม่"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "Ubuntu รุ่นที่คุณใช้ไม่ให้การสนับสนุนอีกต่อไป"

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"สำหรับข้อมูลการปรับรุ่น โปรดดู:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr "การอัปเกรดเป็นรุ่นการพัฒนาสามารถ "

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr "ใช้ได้เฉพาะรุ่นที่รองรับล่าสุดเท่านั้น"

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr "ไม่พบรุ่นใหม่"

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "ยังไม่พร้อมที่จะปรับรุ่นในตอนนี้"

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr "การอัปเกรดรุ่นไม่สามารถทำได้ตอนนี้ โปรดลองใหม่ในภายหลัง เซิร์ฟเวอร์รายงานว่า: '%s'"

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "มี '%s' รุ่นใหม่ที่ใช้ได้"

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "เรียก 'do-release-upgrade' เพื่อที่จะทำการปรับปรุง"

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr "โปรดติดตั้งการอัปเดตทั้งหมดที่พร้อมใช้งานสำหรับรุ่นเผยแพร่ของคุณก่อนทำการอัปเกรด"

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""
"คุณยังไม่ได้บูตระบบใหม่หลังจากที่อัปเดตแพ็กเกจซึ่งต้องบูตระบบใหม่ "
"โปรดบูตระบบใหม่ก่อนทำการอัปเกรดต่อไป"

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr "โปรแกรมอัปเดตซอฟต์แวร์"

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Ubuntu %(version)s สามารถปรับรุ่นได้แล้ว"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "คุณได้ตัดสินใจไม่ปรับเป็นรุ่น Ubuntu  %s"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "ตรวจสอบว่าสามารถปรับปรุงไปสู่รุ่นพัฒนาล่าสุด(latest devel release)ได้หรือไม่"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr "เพิ่มตัวเลือกแสดงกระบวนการแก้ปัญหา"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "มีปัญหาที่แก้ไม่ได้ขณะวางแผนในการปรับปรุง:\n"
#~ "%s\n"
#~ "\n"
#~ " ปัญหาอาจมาจาก:\n"
#~ " * ปรับรุ่นขึ้นไปเป็นรุ่น pre-release ของ Ubuntu\n"
#~ " * ขณะนี้กำลังใช้รุ่น pre-release ของ Ubuntu อยู่\n"
#~ " * ไม่ใช่ซอฟต์แวร์แพคเกจที่มาจาก Ubuntu อย่างเป็นทางการ\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "ระบบของคุณไม่ได้ประกอบด้วย Ubuntu-desktop, Kubuntu-desktop, Xubuntu-desktop "
#~ "หรือ Edubuntu-desktop และไม่สามารถตรวจสอบได้ว่าคุณใช้ Ubuntu รุ่นอะไรอยู่\n"
#~ " กรุณาติดตั้งแพกเกจด้านบนอย่างใดอย่างหนึ่งก่อนโดยใช้โปรแกรม Synaptic หรือ apt-get "
#~ "ก่อนจะดำเนินการต่อไป"

#~ msgid "Sandbox setup failed"
#~ msgstr "การเตรียมกล่องทรายสำหรับทดลองมีปัญหา"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "ไม่สามารถสร้างสภาพแวดล้อมของกล่องทรายได้"

#~ msgid "Sandbox mode"
#~ msgstr "แบบกล่องทรายสำหรับทดลอง"

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "โปรแกรม python ที่คุณติดตั้งไว้ไม่สามารถใช้ได้ กรุณาแก้ไข้ symlink  '/usr/bin/python'"

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "แพคเกจ 'debsig-verify' ถูกติดตั้งไว้แล้ว"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "ไม่สามารถอัปเกรดต่อไปได้ถ้ามีแพคเกจนั้นติดตั้งอยู่.\n"
#~ "โปรดลบมันด้วย synaptic หรือ apt-get remove debsig-verify เสียก่อนแล้วอัปเกรดอีกครั้ง"

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "ในขณะที่สแกนที่คลังข้อมูลของคุณไม่พบรายการที่สำรองสำหรับปรับรุ่น "
#~ "นี้สามารถเกิดขึ้นได้ถ้าคุณเรียกใช้ที่สำรองภายในหรือหากข้อมูลหมดอายุแล้ว\n"
#~ "\n"
#~ "คุณต้องการที่จะเขียนแฟ้ม 'sources.list' ใหม่หรือไม่ หากคุณเลือกที่ 'ใช่' จะทำการปรับปรุง "
#~ "'%s' ทั้งหมดไปเป็น '%s'\n"
#~ "หากคุณเลือก 'ไม่' การปรับรุ่นที่จะยกเลิก"

#~ msgid "evms in use"
#~ msgstr "evms ใช้งานอยู่"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "ระบบของคุณใช้โปรแกรมจัดการโวลุ่ม 'evms' ใน /proc/mounts ซอฟต์แวร์ 'evms' "
#~ "ไม่มีการสนับสนุนแล้ว กรุณาหยุดใช้และทำการปรับปรุงใหม่อีกครั้ง"

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "คอมพิวเตอร์ตัวนี้ใช้กราฟิกไดรเวอร์ของ NVIDIA 'nvidia' "
#~ "ไม่พบรุ่นของไดรเวอร์ที่ทำงานกับการ์ดวิดีโอของคุณใน Ubuntu 10.04 LTS\n"
#~ "\n"
#~ "คุณต้องการทำต่อ?"

#~ msgid "No i686 CPU"
#~ msgstr "ไม่มี CPU i686"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "ระบบของคุณใช้ i586 CPU หรือ CPU ที่ไม่มีส่วนขยาย 'cmov' "
#~ "แพคเกจทั้งหมดถูกสร้างขึ้นด้วยการเพิ่มประสิทธิภาพต้องเป็นสถาปัตยกรรม i686 น้อยที่สุด "
#~ "มันเป็นไปไม่ได้ที่จะปรุบรุ่นระบบ Ubuntu ของคุณกับฮาร์ดแวร์ใหม่นี้"

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "กล่องทรายทดลองปรับปรุงโดยใช้ aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "ทดลองการปรับปรุงด้วย sandbox aufs overlay"

#~ msgid "No new release found"
#~ msgstr "ไม่เจอรุ่นล่าสุด"

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "ถ้าไม่เป็นเพราะปัญหาดังกล่าว โปรดรายงานปัญหานี้โดยใช้คำสั่ง 'ubuntu-bug ubuntu-"
#~ "release-upgrader-core' ในเทอร์มินัล"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "เกิดปัญหาที่แก้ไขไม่ได้ขึ้นขณะกำลังคำนวณการอัปเกรด\n"
#~ "\n"
#~ " อาจเป็นเพราะ:\n"
#~ " * อัปเกรดไปเป็น Ubuntu รุ่นก่อนปล่อยตัวจริง\n"
#~ " * กำลังใช้ Ubuntu ซึ่งรุ่นที่ใช้อยู่เป็นรุ่นก่อนปล่อยตัวจริง\n"
#~ " * แพคเกจซอฟต์แวร์ไม่เป็นทางการที่ไม่ได้จัดให้โดยทีมงาน Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "การอัปเกรดนี้กำลังทำงานในโหมด sandbox (ทดสอบ) การเปลี่ยนแปลงทั้งหมดจะถูกเขียนไปยัง "
#~ "'%s' และจะสูญหายหลังจากเริ่มระบบครั้งต่อไป\n"
#~ "\n"
#~ "*จะไม่มี* "
#~ "การเปลี่ยนแปลงใดๆถูกเขียนไปยังไดเรกทอรีของระบบจากตอนนี้จนกระทั่งเริ่มระบบครั้งต่อไปอย่างถาวร"

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "การอัปเกรดถูกยกเลิก การอัปเกรดต้องการพื้นที่ว่างในดิสก์ '%s' รวมทั้งหมด '%s' "
#~ "โปรดเพิ่มพื้นที่ว่างบน '%s' อย่างน้อย %s "
#~ "ลบข้อมูลทั้งหมดในถังขยะของคุณและถอดถอนแพกเกจชั่วคราวที่ติดตั้งเมื่อครั้งที่ผ่านมาโดยใช้คำสั่ง "
#~ "'sudo apt-get clean'"

#~ msgid "apt architecture is i386"
#~ msgstr "สถาปัตยกรรม apt คือ i386"

#~ msgid ""
#~ "Upgrades from 18.04 on the i386 architecture are not supported at this "
#~ "time."
#~ msgstr "ไม่รองรับการอัปเกรดจากรุ่น 18.04 บนสถาปัตยกรรม i386 ในขณะนี้"
